stripe-payment-link-sms/assets/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <style>
+      body {
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+      }
+
+      div[role="main"] {
+        flex: 1;
+      }
+
+      footer {
+        text-align: center;
+      }
+
+      footer p {
+        margin-bottom: 0;
+      }
+
+      #twilio-logo {
+        width: 50px;
+        height: 50px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to your new Twilio App</h1>
+    </header>
+    <div role="main">
+      <section>
+        <h3>Get started with your app</h3>
+        <p>This app shows you how to create an invoice in Stripe based on an SMS prompt (in this case `DONATE <AMOUNT>`). Once the invoice is finalized, it will reply with a payment link.</p>
+        <ol>
+          <li>Before you can test your app, you need to add a <a href="https://dashboard.stripe.com/webhooks"> Stripe Webhook</a> and set the URL to the URL of this page, but replace `/index.html` with `/send-invoice-sms`</li>
+          <li>Select "All Events" and save the webhook.</li>
+          <li>To test the customer initiated scenario of this app, text `DONATE 10` to your Twilio phone number referenced below. You should receive a Stripe invoice link to your phone.</li>
+          <li>To test the Merchant initiated flow, you need to create an invoice in the Stripe Dashbboard. Follow this <a href="https://stripe.com/docs/billing/invoices/create#without-code" guide</a> or this <a href="https://youtu.be/YftvRXupgpw" video </a>. Make sure to set a phone number when entering the customer details.</li>
```

Comment:
`<a href="https://stripe.com/docs/billing/invoices/create#without-code" guide</a>` should be `<a href="https://stripe.com/docs/billing/invoices/create#without-code">guide</a>`

`<a href="https://youtu.be/YftvRXupgpw" video </a>.` should be `<a href="https://youtu.be/YftvRXupgpw">video</a>.`